source/main.cpp

```diff
@@ -29,8 +29,69 @@
 
 using namespace android;
 
+// Pick one
+// #define WANT_APPLET
+#define WANT_SYSMODULE
```

Comment:
Default to applet

Reply:
Oof. I changed that and then wanted to test the try catch fix and forgot to change back again lol